beluga/CMakeLists.txt

```diff
@@ -25,6 +25,18 @@ if(NOT CMAKE_BUILD_TYPE)
       CACHE STRING "Build type" FORCE)
 endif()
 
+# Enable assertions in RelWithDebInfo builds by removing the NDEBUG definition.
```

Comment:
I followed this approach instead of creating a new build type so that we can pass the same argument to all packages when we run colcon build.

Reply:
Why not build both for Debug and Release instead? Speed? Cost?

Reply:
Speed ​​is the main factor, Eigen, range-v3 and our own code rely heavily on a basic level of optimization to run reasonably fast.

Reply:
Hmm, actually some tests fail in debug builds. Let me dig into that.